src/main/CommandLine.cpp

```diff
@@ -1360,6 +1360,65 @@ getSettingsUpgradeTransactions(CommandLineArgs const& args)
         });
 }
 
+int
+runCheckQuorumIntersection(CommandLineArgs const& args)
+{
+    CommandLine::ConfigOption configOption;
+    std::string jsonPath;
+    return runWithHelp(
+        args,
+        {logLevelParser(configOption.mLogLevel), fileNameParser(jsonPath),
+         clara::Opt{configOption.mConfigFile,
+                    "FILE-NAME"}["--conf"](fmt::format(
+             FMT_STRING("specify a config file to enable human readable "
+                        "node names (optional, '{}' for STDIN)"),
+             Config::STDIN_SPECIAL_NAME))},
+        [&] {
+            try
+            {
+                std::optional<Config> cfg = std::nullopt;
+                if (configOption.mConfigFile.empty())
+                {
+                    // Need to set up logging in this case because there is no
+                    // `getConfig` call (which would otherwise set up logging)
+                    Logging::setLoggingToConsole(true);
+                    Logging::setLogLevel(configOption.mLogLevel, nullptr);
+                }
+                else
+                {
+                    cfg.emplace(configOption.getConfig(false));
```

Comment:
why not log to file?

Reply:
This is a holdover from when config files were mandatory and I thought console logging was more practical for this use case (otherwise there's very little output, and the log is tiny anyway). Now that the config file is optional I agree the command should just interpret the config file as-is. I changed this to log to a file and added a `--console` flag for those who want console logging. (see 7ebfcf096e2a971e448fc6a4b80b69fd259b1440)